### DISCUSSION TITLE
Intensity should be measured at a specified and recorded time and place

### DISCUSSION THREAD

**Discussion by adrianco** [Green-Software-Foundation/sci#330] [SCI feedback]
Carbon Intensity varies, so when an SCI value is reported, it should be labeled with the time and place that the Intensity was obtained for. In most cases, a month and a region will be sufficient (January 2022, us-east-1) but where 24x7 data is available, the time of day in local timezone is needed (10:00-10:59am 10th January 2022 us-east-1). This allows normalization of SCI comparisons, and allows for older data to be recognized and disregarded as the grids decarbonize over time.

**Comment by jawache** [comment]
Good point. We haven't got consensus yet on the roadmap past 1.0, but I do remember the group had decided to not include reporting standards as part of 1.0 and to cover that in a future iteration. The time/space question is a good point we should make sure to add in, we've got a few other knarly reporting issues we need to address as well, like how to report he software boundary, how to baseline etc...

@Henry-WattTime / @atg-abhishek do we have labels for issues/discussions to add to the roadmap for discussion post 1.0? Or maybe we can add an agenda item once 1.0 is official to discuss roadmap for future things we want the spec to cover.

**Reply by atg-abhishek** [reply]
I think now would be a good time to think about this, I've added a new label called `sci-update` for capturing these kinds of discussions. Perhaps something that we can discuss in the upcoming meeting? cc @seanmcilroy29

**Comment by Henry-WattTime** [comment]
WG: 
Henry: Important point on how to include timeline
Sara: There may be people who do not want to share region/time data, if we require reporting, will it be an obstacle to folks reporting? Maybe an optional field? This can be very sensitive information (security, business, etc).
Gadhu: Guideline for people measuring internally, even if not reported publicly
Conclusion: we should carefully consider whether this can be included in following version, based on potential limitations
